plop/hook/tests/{{camelCase hookName}}.test.ts.hbs

```diff
@@ -0,0 +1,9 @@
+import { renderHook } from "@infinum/test-utils";
```

Comment:
Hehe we need to create this lib too `@infinum/test-utils`

Reply:
It's my leftover

Reply:
also, it should be named `react-test-utils`... or should we have a specific org like `@infinum-react/`